metadata/contributors.json

```diff
@@ -756,7 +756,7 @@
    },
    {
       "emails" : [
-         "aizquierdogarcia@apple.com"
+         "a_izquierdogarcia@apple.com"
```

Comment:
My suggestion is to keep your old email as the second element in the list (I gave @Smackteo the same advice yesterday)

TLDR, if you keep your old email, tools will use your new email when printing information about your old commits.